src/quantuminspire/api.py

```diff
@@ -676,6 +676,32 @@ def get_measurement_register_from_result(self, result_id: int) -> List[Any]:
             raise ApiError(f'Measurement register for result with id {result_id} does not exist!') from err_msg
         return measurement_register
 
+    def get_calibration_from_result(self, result_id: int) -> Optional[Any]:
+        """ Gets the calibration information of the executed cQASM code, given the result_id.
+
+        :param result_id: The identification number of the result.
+
+        :raises ApiError: If the calibration url in result is invalid or the request
+            for the calibration info using the url failed.
+
+        :return:
+            The calibration information from the backend.
+            None is returned when there is no calibration information (e.g. simulator backend).
+        """
+        calibration_info = None
+        result = self.get_result(result_id)
+        calibration_url = result.get('calibration')
+        if calibration_url is not None:
+            try:
```

Comment:
I guess this code works, but is the parsing of the url needed? One can just do
```
result = requests.get(c, auth=authentication)
calibration = result.json()
```
(the `authentication` and `.json` can be wrapped in something off course)

Reply:
Yes, you can directly use request package to send a get for this url,, but I think it is better to use the already authenticated coreapi consistently for all communication with the backend api as it is now done in SDK.